crates/apollo-mcp-server/src/operations/schema_walker.rs

```diff
@@ -0,0 +1,43 @@
+//! JSON Schema generation utilities
+//!
+//! The types in this module generate JSON schemas for GraphQL types by walking
+//! the types recursively.
+
+use apollo_compiler::{Schema as GraphQLSchema, ast::Type};
+use schemars::Schema;
+use serde_json::{Map, Value};
+
+use crate::custom_scalar_map::CustomScalarMap;
+
+pub(super) mod name;
+pub(super) mod r#type;
```

Comment:
These could just be private since they're not used outside this module.

```suggestion
mod name;
mod r#type;
```